apps/designer/app/designer/features/style-panel/shared/parse-css-value.ts

```diff
@@ -22,31 +22,36 @@ export const isValid = (property: string, value: string): boolean => {
   return true;
 };
 
-// wtf?
-// eslint-disable-next-line no-useless-escape
-const mathRegex = /[\+\-\*\/]/;
+// If expression is a math expression, evaluates it.
+// Otherwise returns undefined.
+const evaluateMath = (expression: string) => {
+  if (/^\d[+\-*/0-9.]+\d$/.test(expression) === false) {
```

Comment:
0.5 + 0.5 not covered, yes?

Reply:
IMO be simple `^[\d.+\-*/\s]+$` just check that only numbers +-/* and space exists